models/ssd_detector/regression_test/train_net.py

```diff
@@ -1,224 +0,0 @@
-import datetime
-import json
-import multiprocessing as mp
-import os
-import shutil
-import unittest
-
```

Comment:
Looks like you delete this file. Why not move? It is better for history tracking.